src/tests/main.cpp

```diff
@@ -71,6 +75,7 @@ struct wesnoth_global_fixture {
 		}
 
 		reporter.open(file.c_str());
```

Comment:
Is it also fixed by changing this line to use `file.string()` instead of `file.c_str()`?

Reply:
it done